resources/views/components/nav.blade.php

```diff
@@ -8,13 +8,13 @@
                 <span class="icon-bar"></span>
             </button>
 
-            <a class="navbar-brand" href="/">Chompy</a>
+            <a class="navbar-brand">Chompy</a>
         </div>
         <div id="navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav navbar-right">
                 @if (Auth::user())
-                    <li class="{{ request()->input('type') === \Chompy\ImportType::$rockTheVote ? 'active' : '' }}">
-                        <a class="nav-item nav-link" href="/import?type={{ \Chompy\ImportType::$rockTheVote }}">
+                    <li class="{{ str_contains(request()->url(), $rockTheVote) ? 'active' : '' }}">
```

Comment:
Doing a string comparison on the whole URL feels like it could result in subtle bugs (e.g. if a future import is a substring of another one, or shows up in an unrelated query-string). Let's try matching on the path instead:

```blade
<li class="{{ request()->is('import/' . $rockTheVote) ? 'active' : '' }}">
```